run-test.py

```diff
@@ -218,12 +256,16 @@ def k8s():
 
 def k8s_gen_yaml(fname, image_name, pull_policy):
     with open(fname, "w") as f:
+        f.write(K8S_YAML_TOOLS_TEMPLATE.format(image_name=image_name, pull_policy=pull_policy))
         for i in ALL_PODS:
-            pod_name = "jepsen-"+str(i)
+            pod_name = "jepsen-" + str(i)
             ssh_port = str(32000 + i)
             descr = K8S_YAML_TEMPLATE.format(
-                pod_name = pod_name, ssh_port = ssh_port,
-                image_name = image_name, pull_policy = pull_policy)
+                pod_name=pod_name,
```

Comment:
We need to run reformat for all .py files, 
tabs and whitespaces in one file detected %)

Reply:
Sure, feel free to do it in this PR.